### DISCUSSION TITLE
Feature-request: word count per tag

### DISCUSSION THREAD

**Discussion by grero** [standardnotes/forum#2609] [Suggestions]
I am experimenting with using Standard Notes as a substitute for writing-specific Apps such as Ulysses. One feature that would be really useful to have is a way track word count per tag. This can probably be implemented via an extension, and I wouldn't mind giving that a go, but I wanted to check whether something like his already exists/is in the works. Thanks!

**Comment by effieeee** [comment]
That's quite an interesting feature to add, and yes, perhaps it would need a separate extension, or an amendment to the Action Bar. I believe nothing like this is currently in the works, though this is something we can keep in mind!

**Comment by myreli** [comment]
Closing to track on https://github.com/standardnotes/forum/discussions/2675.
